src/components/Row/index.tsx

```diff
@@ -18,7 +26,7 @@ const Row = styled(Box)<{
   padding: ${({ padding }) => padding};
   border: ${({ border }) => border};
   border-radius: ${({ borderRadius }) => borderRadius};
-  gap: ${({ gap }) => gap};
+  gap: ${({ gap, theme }) => gap && (theme.grids[gap as Gap] || gap)};
```

Comment:
since none of the other props here are theme tokens thoughts on just letting the consumer figure out the right value?

Reply:
For the `gap`? The primary motivation was to align with `Column`, which already has a `gap` property that accepts theme tokens. Not sure why this wasn't available for `Row` yet.

Reply:
omg 😂